V1Documentation.md

```diff
@@ -199,20 +163,20 @@ objects (for partial matches). Each should match the hostname or TLD that you wa
 An example:
 
 ```javascript
-rg4js('options', {
+Raygun.init('apikey', {
   allowInsecureSubmissions: true,
   ignoreAjaxAbort: true,
   ignoreAjaxError: true,
   debugMode: true,
   ignore3rdPartyErrors: false,
   wrapAsynchronousCallbacks: true,
-  excludedHostnames: ['\.local'],
-  excludedUserAgents: ['Mosaic'],
-  disableCrashReporting: false,
+  excludedHostnames: ['localhost', '\.dev'],
+  excludedUserAgents: ['PhantomJS', 'MSIE'],
+  disableErrorTracking: false,
   disablePulse: false,
-  pulseMaxVirtualPageDuration: 1800000,
+  pulseMaxVirtualPageDuration: 1800000
   pulseIgnoreUrlCasing: false
-});
+}).attach();
```

Comment:
.attach should be on a newline with a comment after it to highlight the fact that it enables CrashReporting